client/app/components/project-geography.hbs

```diff
@@ -16,24 +16,26 @@
 	  <legend data-test-development-site-question="{{bbl.dcpBblnumber}}-{{bbl.dcpDevelopmentsite}}">
 	    Is this BBL part of the development site?
 	  </legend>
-	  <input
-		type="radio"
+	  <Input
+		@type="radio"
 		name={{concat idx '-bbl-dev-site'}}
 		id={{concat idx '-bbl-dev-site-yes'}}
-		data-test-bbl-development-site-yes="{{bbl.dcpBblnumber}}"
-		checked={{if bbl.dcpDevelopmentsite "checked"}}
+		@value={{eq bbl.dcpDevelopmentsite true}}
+		@checked={{eq bbl.dcpDevelopmentsite true}}
 		onClick={{fn this.updateAttr bbl "dcpDevelopmentsite" true}}
+		data-test-bbl-development-site-yes="{{bbl.dcpBblnumber}}"
 	  />
```

Comment:
Yeah, I noticed you are keeping the indentation flat here. Why? It's also still tab indentation. We use 2-space indentation, not tab.

Reply:
I have been using 2-space, what is indicating tab spacing to you? What do you mean by flat?

Reply:
It's showing up different because you're using tabs instead of spaces. You have to configure your VS code editor:

<img width="801" alt="Screen Shot 2020-04-30 at 1 54 30 PM" src="https://user-images.githubusercontent.com/5004319/80742934-2bcc2180-8aea-11ea-82bf-c86449ef709d.png">

It should use 2 spaces.

Reply:
When I say flat, I mean this looks wrong:

<img width="303" alt="Screen Shot 2020-04-30 at 1 58 36 PM" src="https://user-images.githubusercontent.com/5004319/80743271-b1e86800-8aea-11ea-83d9-5bbf5698560d.png">

The Input attrs should be indented more. This is how all of our other templates look.

It should be